### PR TITLE
Support manager job

### PR DESCRIPTION
**Pull request by GShotwell**
I did a fair amount of editing to the original job description mostly to make it a bit more succinct.

### CODE REVIEW

File: content/jobs/support-developer.md [hunk 1]
```diff
@@ -8,23 +8,38 @@ images = ["https://crunch.io/img/logo-1200x630.png"]
...
-* Field user questions and reports and triage them appropriately, including identifying clear steps to reproduce issues; contribute to the design of a scalable customer-support system
-* Write public resources (documentation, examples, blog posts) that educate current and potential users on the features of our product and how to most effectively use them
-* Work with product management to refine feature specifications based on your support interactions
-* Collaborate with engineering leads to schedule work on urgent and/or important user-reported issues
-* Draft content and communicate with users, both individually to notify them that requested features/bugs have been addressed, and collectively through in-app announcements, release notices, and mass emails.
+* Provide friendly first line communication for customers 
+* Develop user-training materials and documentation
+* Identify and replicate bugs with the software and help our development team understand and prioritize resolutions
+* Take ownership over the customer support to ensure that none of our users fall through the cracks
+* Analyze customer behavior to Identify and prioritize areas of the app which need improvement
```
Comment (jonkeane) [inline content/jobs/support-developer.md]:
```suggestion
* Analyze customer behavior to identify and prioritize areas of the app which need improvement
```

File: content/jobs/support-developer.md [hunk 2]
```diff
@@ -8,23 +8,38 @@ images = ["https://crunch.io/img/logo-1200x630.png"]
...
+* Help publicize and communicate new features to customers
```
Comment (jonkeane) [inline content/jobs/support-developer.md]:
```suggestion
* Help publicize and communicate new features to customers by writing blog posts, user-friendly release announcements, etc.
```

File: content/jobs/support-developer.md [hunk 3]
```diff
@@ -8,23 +8,38 @@ images = ["https://crunch.io/img/logo-1200x630.png"]
...
-* Attention to detail
-* Strong analytical and deductive skills, important for isolating the cause of bugs
-* Excellent writing ability: you can express yourself clearly and with appropriate tone and grammar
-* Empathy, both for users and engineers
-* Programming skills, or an interest in developing programming skills, are a plus
-* Strong organizational skills and ability to field multiple concurrent responsibilities without letting any slip through
+Required skills:
+
+* 2 - 5 years experience in market research, survey analysis, or customer support
+* Bachelor’s degree in a quantitative field, or equivalent experience
```
Comment (jonkeane) [inline content/jobs/support-developer.md]: I'm not a fan of putting a BA degree here unless we absolutely need it (and in this position we do not). It discourages people who might not have one to apply even with "or equivalent experience". If someone doesn't have the experience we will see that.

File: content/jobs/support-developer.md [hunk 4]
```diff
@@ -8,23 +8,38 @@ images = ["https://crunch.io/img/logo-1200x630.png"]
...
+* Excellent verbal and written communication skills, including the ability to clearly explain complex topics to a variety of audiences
+* Self-motivated and able to manage and prioritize your work
+* Experience providing technical support to system administrators, developers, or data scientists
```
Comment (jonkeane) [inline content/jobs/support-developer.md]: I'm not 100% for sure what this is is looking for, is it possible we could reword it?

File: content/jobs/support-developer.md [hunk 5]
```diff
@@ -8,23 +8,38 @@ images = ["https://crunch.io/img/logo-1200x630.png"]
...
+* Unflagging empathy for users
```
Comment (jonkeane) [inline content/jobs/support-developer.md]: Probably this is me / our terminology but maybe unflagging > unwavering? I read unflagging and think it's about our feature flagging processes

File: content/jobs/support-developer.md [hunk 6]
```diff
@@ -8,23 +8,38 @@ images = ["https://crunch.io/img/logo-1200x630.png"]
...
+
+In addition we prefer experience in:
```
Comment (jonkeane) [inline content/jobs/support-developer.md]:
```suggestion
Nice to have, but not required experience:
```

File: content/jobs/support-developer.md (hunk 5 above)
Comment (GShotwell) [reply content/jobs/support-developer.md]: Oh yeah that's much better. I've been reading too many naval warfare in the age of sail books.

File: content/jobs/support-developer.md [hunk 7]
```diff
@@ -8,23 +8,38 @@ images = ["https://crunch.io/img/logo-1200x630.png"]
...
+* 2 - 5 years experience in market research, survey analysis, or customer support
+* Bachelor’s degree in a quantitative field, or equivalent experience
+* Excellent verbal and written communication skills, including the ability to clearly explain complex topics to a variety of audiences
+* Self-motivated and able to manage and prioritize your work
+* Experience providing technical support to system administrators, developers, or data scientists
+* Unflagging empathy for users
+
+In addition we prefer experience in: 
+
+* Programming, ideally in R or Python
```
Comment (jonkeane) [inline content/jobs/support-developer.md]: I like the original hedge we had here: `Programming skills, or an interest in developing programming skills, are a plus` a good communicator with only an interest in programming >>> a bad communicator, and being experience in R/Python in this position IMO

File: content/jobs/support-developer.md (hunk 3 above)
Comment (GShotwell) [reply content/jobs/support-developer.md]: I'm happy either way, my guess is that we're going to get an overwhelming number of applicants even with the degree requirement

File: content/jobs/support-developer.md (hunk 4 above)
Comment (GShotwell) [reply content/jobs/support-developer.md]: Hmmm, reading it again I think we should just take it out

File: content/jobs/support-developer.md [hunk 8]
```diff
@@ -1,30 +1,41 @@
...
+Crunch.io is an innovative analytics company in the market research industry. We offer a survey data analysis platform that allows market researchers, analysts, and marketers to collaborate in a secure, cloud-based environment, using an intuitive drag-and-drop interface. Crunch users can quickly gain insights from their data and communicate them to their clients. We are hiring a Customer Experience Manager to help us better understand our customers and to improve the product to meet their needs.
```
Comment (jonkeane) [inline content/jobs/support-developer.md]:
```suggestion
Crunch.io is an innovative analytics company in the market research industry. We offer a survey data analysis platform that allows market researchers, analysts, and marketers to collaborate in a secure, cloud-based environment, using an intuitive drag-and-drop interface. Crunch users can quickly gain insights from their data and communicate them to their clients. We are hiring a Customer Experience Manager to help us better support our customers, to help us better understand our customers, and to improve the product to meet their needs.
```

**Comment by nealrichardson** [comment]
I pseudo-cherry-picked/squashed the PR and put it on `src` here https://github.com/Crunch-io/ta-da/commit/a733969f1213736246ad1f724437b06ec088f363 because this branch was based off `dev` (otherwise LGTM).

**Pull request closed without merging**